javalib/src/main/scala/java/lang/Character.scala

```diff
@@ -1371,189 +1371,433 @@ object Character {
+      new UnicodeBlock("OTTOMAN_SIYAQ_NUMBERS", 0x1ED00, 0x1ED4F)
+    val ARABIC_MATHEMATICAL_ALPHABETIC_SYMBOLS = new UnicodeBlock(
+      "ARABIC_MATHEMATICAL_ALPHABETIC_SYMBOLS",
+      0x1EE00,
+      0x1EEFF)
+    val MAHJONG_TILES = new UnicodeBlock("MAHJONG_TILES", 0x1F000, 0x1F02F)
+    val DOMINO_TILES  = new UnicodeBlock("DOMINO_TILES", 0x1F030, 0x1F09F)
+    val PLAYING_CARDS = new UnicodeBlock("PLAYING_CARDS", 0x1F0A0, 0x1F0FF)
+    val ENCLOSED_ALPHANUMERIC_SUPPLEMENT =
+      new UnicodeBlock("ENCLOSED_ALPHANUMERIC_SUPPLEMENT", 0x1F100, 0x1F1FF)
+    val ENCLOSED_IDEOGRAPHIC_SUPPLEMENT =
+      new UnicodeBlock("ENCLOSED_IDEOGRAPHIC_SUPPLEMENT", 0x1F200, 0x1F2FF)
+    val EMOTICONS = new UnicodeBlock("EMOTICONS", 0x1F600, 0x1F64F)
+    val ORNAMENTAL_DINGBATS =
+      new UnicodeBlock("ORNAMENTAL_DINGBATS", 0x1F650, 0x1F67F)
+    val ANATOLIAN_HIEROGLYPHS =
+      new UnicodeBlock("ANATOLIAN_HIEROGLYPHS", 0x1F700, 0x1F77F)
+    val GEOMETRIC_SHAPES_EXTENDED =
+      new UnicodeBlock("GEOMETRIC_SHAPES_EXTENDED", 0x1F780, 0x1F7FF)
+    val SUPPLEMENTAL_ARROWS_C =
+      new UnicodeBlock("SUPPLEMENTAL_ARROWS_C", 0x1F800, 0x1F8FF)
+    val CHESS_SYMBOLS = new UnicodeBlock("CHESS_SYMBOLS", 0x1FA00, 0x1FA6F)
     val CJK_UNIFIED_IDEOGRAPHS_EXTENSION_B =
-      new UnicodeBlock("CJK_UNIFIED_IDEOGRAPHS_EXTENSION_B", 0x20000, 0x2a6df)
+      new UnicodeBlock("CJK_UNIFIED_IDEOGRAPHS_EXTENSION_B", 0x20000, 0x2A6DF)
+    val CJK_UNIFIED_IDEOGRAPHS_EXTENSION_C =
+      new UnicodeBlock("CJK_UNIFIED_IDEOGRAPHS_EXTENSION_C", 0x2A700, 0x2B73F)
+    val CJK_UNIFIED_IDEOGRAPHS_EXTENSION_D =
+      new UnicodeBlock("CJK_UNIFIED_IDEOGRAPHS_EXTENSION_D", 0x2B740, 0x2B81F)
+    val CJK_UNIFIED_IDEOGRAPHS_EXTENSION_E =
+      new UnicodeBlock("CJK_UNIFIED_IDEOGRAPHS_EXTENSION_E", 0x2B820, 0x2CEAF)
+    val CJK_UNIFIED_IDEOGRAPHS_EXTENSION_F =
+      new UnicodeBlock("CJK_UNIFIED_IDEOGRAPHS_EXTENSION_F", 0x2CEB0, 0x2EBEF)
     val CJK_COMPATIBILITY_IDEOGRAPHS_SUPPLEMENT = new UnicodeBlock(
       "CJK_COMPATIBILITY_IDEOGRAPHS_SUPPLEMENT",
-      0x2f800,
-      0x2fa1f)
-    val TAGS = new UnicodeBlock("TAGS", 0xe0000, 0xe007f)
+      0x2F800,
+      0x2FA1F)
+    val TAGS = new UnicodeBlock("TAGS", 0xE0000, 0xE007F)
     val VARIATION_SELECTORS_SUPPLEMENT =
-      new UnicodeBlock("VARIATION_SELECTORS_SUPPLEMENT", 0xe0100, 0xe01ef)
+      new UnicodeBlock("VARIATION_SELECTORS_SUPPLEMENT", 0xE0100, 0xE01EF)
     val SUPPLEMENTARY_PRIVATE_USE_AREA_A =
-      new UnicodeBlock("SUPPLEMENTARY_PRIVATE_USE_AREA_A", 0xf0000, 0xfffff)
+      new UnicodeBlock("SUPPLEMENTARY_PRIVATE_USE_AREA_A", 0xF0000, 0xFFFFF)
     val SUPPLEMENTARY_PRIVATE_USE_AREA_B =
-      new UnicodeBlock("SUPPLEMENTARY_PRIVATE_USE_AREA_B", 0x100000, 0x10ffff)
+      new UnicodeBlock("SUPPLEMENTARY_PRIVATE_USE_AREA_B", 0x100000, 0x10FFFF)
+    // scalastyle:on line.size.limit
 
-    private val BLOCKS = Array(
+    private val allBlocks: Array[UnicodeBlock] = Array(
```

Comment:
Now this is the type of thing that needs to be lazy, if possible.  I did not trace how it gets used.  This table will, I believe, cause all of the UnicodeBlocks it references to be
instantiated.

Reply:
May need to be an method which returns the "known&fast" UnicodeBlocks, then iterates over a lazy array of "complete&slow" UnicodeBlocks.